src/rename.js

```diff
@@ -2,10 +2,12 @@ const assoc    = require('ramda/src/assoc')
 const converge = require('ramda/src/converge')
 const curry    = require('ramda/src/curry')
 const dissoc   = require('ramda/src/dissoc')
+const hasIn    = require('ramda/src/hasIn')
 const prop     = require('ramda/src/prop')
+const when     = require('ramda/src/when')
 
 // rename :: String -> String -> { k: v } -> { k: v }
-const rename = (from, to, obj) =>
-  converge(assoc(to), [ prop(from), dissoc(from) ])(obj)
+const rename = (frum, to, obj) =>
```

Comment:
ok, I'll bite, why `frum`?

Reply:
oh, because `from` is a reserved word in es6.

Reply:
yep.  even my highlighter colors it as a reserved word.  plus, `frum` makes your face look poutier when you say it.